chpl/chpl-service/src/main/java/gov/healthit/chpl/upload/certifiedProduct/CertifiedProductHandler2014.java

```diff
@@ -699,7 +699,7 @@ private void parseTestTools(PendingCertifiedProductEntity product, PendingCertif
 				ttEntity.setTestToolVersion(testToolVersion);
 				TestToolDTO testTool = testToolDao.getByName(testToolName);
 				if(testTool != null) {
-					if(testTool.isRetired() && product.getIcs().equals("false")) {
+					if(testTool.isRetired() && product.getIcs() == false) {

```

Comment:
Got it! As discussed in slack, I think this logic actually needs to be moved into one of the validator classes. Leaving as changes requested for now.